charts/federation-v2/charts/clusterregistry/templates/namespace.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: {{ .Values.global.registryNamespace }}
```

Comment:
For namespaced deployment, there is no need to create this namespace because cluster registry are in the same namespace with federation